dropwizard-http2/src/main/java/io/dropwizard/http2/Http2ConnectorFactory.java

```diff
@@ -123,4 +130,10 @@ public Connector build(Server server, MetricRegistry metrics, String name, Threa
                 new Jetty93InstrumentedConnectionFactory(sslConnectionFactory, metrics.timer(httpConnections())),
                 alpn, http2, http1);
     }
+
+    private static List<String> addIfNotPresent(List<String> someList, String elementToAddIfNotPresent) {
```

Comment:
I would rename `someList` to `sourceList` for clarity and `elementToAddIfNotPresent` to `addedElement` for terseness.